docs/guides/chip_tool_guide.md

```diff
@@ -868,3 +868,97 @@ In this command:
 -   _<node_id\>_ is the ID of the node that is going to receive the binding.
 -   _<endpoint_id\>_ is the ID of the endpoint on which the `binding` cluster is
     implemented.
+
+### Multi-admin scenario
+
+Multi-admin feature allows you to join Matter device to several Matter fabrics
+and administer it by several different Matter administrators. First you need to
+commission the Matter device to first fabric following the
+[Using CHIP Tool for Matter device testing](#using) section.
+
+Before it is possible to commission a Matter device to a new fabric, the
+administrator from first fabric must open the commissioning window for a new
+administrator from another fabric.
+
+To open the commissioning window on the paired Matter device, use the following
+command pattern:
+
+```
+$ ./chip-tool pairing open-commissioning-window <node_id> <option> <window_timeout> <iteration> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the ID of the node that should open commissioning window
+-   _<option\>_ is equal to 1 for Enhanced Commissioning Method and 0 for Basic
+    Commissioning Method
+-   _<window_timeout\>_ is time in seconds, before the commissioning window
+    closes
+-   _<iteration\>_ is number of PBKDF iterations to use to derive the PAKE
+    verifier
+-   _<discriminator\>_ is device specific value determined during commissioning
+
+**Example of command:**
+
+```
+$ ./chip-tool pairing open-commissioning-window 1 1 300 1000 3840
+```
+
+Note manual pairing code printed in the command output, as it will be required
+to use by the second Matter admin to join Matter device to its fabric.
+
+**Example of output:**
+
+```bash
+[1663675289.149337][56387:56392] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_003C Command=0x0000_0000 Status=0x0
+
+[1663675289.149356][56387:56392] CHIP:CTL: Successfully opened pairing window on the device
+
+[1663675289.149409][56387:56392] CHIP:CTL: Manual pairing code: [36310944684]
+```
+
+To commission the Matter device to a new fabric, you need to run another
+instance of CHIPTool, using the following command pattern:
```

Comment:
```suggestion
instance of CHIP Tool, using the following command pattern:
```
@markaj-nordic had this comment already but I accidentally deleted the wrong comment.... :(